### ISSUE TITLE
live for <duration> announcements should not be in aria live region

### ISSUE THREAD

**Issue by albertotirla** [owncast/owncast#3334]
## type

bug

## owncast version/tag:

0.1.1

## description

the live for XXX event fires in the aria live region every time the responsible component changes in the DOM. While this is very often in the first parts of the stream, only to ramp down as the stream progresses, this can get annoying very quickly because the screenreader announces every one of those changes individually.

To make a comparison for a second, imagine a box in the top-right corner of the screen flashing a bright red every couple seconds, distracting you from the chat or whatever the stream is about. While, say, the distance between the flashes increases, it is not an ideal user experience at all, it feels the same way to us, but worse, because it interrupts what the screenreader was saying before the announcement.

More important things should be in the aria live region instead, for example new messages, status events some of which are already there, but that's for another issue.

## expected behaviour

that information is not announced at all, because the element is visible in the DOM and we can inspect it, if we want to know for how long the stream has been ongoing

## steps to reproduce

* turn on a screenreader. For example, narrator for windows, orca for linux, voiceover for ios, or talkback for android
* open a browser to an owncast stream. It doesn't matter if you or someone else is streamming it, but preferable here is that the stream is in its early beginning, before the one hour mark, because then the effect is more observable
* just stay there for afew seconds, you should start getting spammed with live for xxx minutes and yyy seconds, one or so per second

## possible solutions

remove that entirely from the aria live region, we don't need that information live like that, there are better things we would rather be spammed with

**Comment by armadi1809** [comment]
@gabek If this one is still really open I can take a shot at it.

**Comment by gabek** [comment]
@armadi1809 It is, thank you!
